eternalcore-core/src/main/java/com/eternalcode/core/feature/chat/ChatManagerCommand.java

```diff
@@ -76,24 +77,42 @@ void disable(Viewer viewer, CommandSender sender) {
             .send();
     }
 
-    @Execute(route = "slowmode", required = 1)
-    @DescriptionDocs(description = "Sets slowmode for chat", arguments = "<time>")
-    void slowmode(Viewer viewer, @Arg Duration duration) {
+    @Execute(name = "slowmode")
+    @DescriptionDocs(description = "Sets SlowMode for chat", arguments = "<time>")
```

Comment:
nie trzeba aż tak kapitalizować tego `SlowMode`, to nie jest jakoś specjalnie nazwa własna, ale jak chcesz
```suggestion
    @DescriptionDocs(description = "Sets slowmode for chat", arguments = "<time>")
```